docker/docker_instructions_ov2.0.md

```diff
@@ -0,0 +1,157 @@
+# Run Docker Images For ROS2_OpenVINO_Toolkit
+
+**NOTE:**
+Below steps have been tested on **Ubuntu 20.04**.
+
+## 1. Environment Setup
+* Install docker ([guide](https://docs.docker.com/engine/install/ubuntu/))
+
+## 2. Build docker image by dockerfile
+```
+cd ~/ros2_openvino_toolkit/docker/Dockerfile
+vi ~/ros2_openvino_toolkit/docker/Dockerfile
+docker build --build-arg ROS_VERSION=<EXPECT_ROS_BASE_IMAGE> --build-arg VERSION=<EXPECT_ROS_VERSION> -t ros2_openvino_202201 .
+```
+For example:
+* Build image for ros_galactic
+```
+cd ~/ros2_openvino_toolkit/docker/Dockerfile
+vi ~/ros2_openvino_toolkit/docker/Dockerfile
+docker build --build-arg ROS_VERSION=galactic-desktop --build-arg VERSION=galactic -t ros2_galactic_openvino_202201 .
+```
+* Build image for ros_foxy
+```
+cd ~/ros2_openvino_toolkit/docker/Dockerfile
+vi ~/ros2_openvino_toolkit/docker/Dockerfile
+docker build --build-arg ROS_VERSION=foxy-desktop --build-arg VERSION=foxy -t ros2_foxy_openvino_202201 .
+```
+**Note**:Proxy may need to set, please check dockerfile before building.
+
+## 3. Download and load docker image
+* Download docker image
+```
+ # ros2_openvino_202201 for demo
+ cd ~/Downloads/
+ wget <DOCKER_IMAGE_PATH>
+```
+* Load docker image
+```
+cd ~/Downloads/
+docker load -i <DOCKER_IMAGE>
+docker images
+/// (show <DOCKER_IMAGE> in the list)
+```
+
+## 4. Running the Demos
+* Install dependency
+```
+  sudo apt install x11-xserver-utils
+  xhost +
+```
+* run docker image
+```
+  docker images
+  docker run -itd  -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix -v /dev:/dev  --privileged=true --name <your_image_name> <IMAGE_ID>
+```
+* In Docker Container
+
+* Preparation
+```
+source /opt/intel/openvino_2022/setupvars.sh
+source /opt/ros/<ROS_VERSION>/setup.bash
+cd ~/catkin_ws
+source ./install/local_setup.bash
+```
+
+* See all available models
+OMZ tools are provided for downloading and converting OMZ models in ov2022.([guide](https://pypi.org/project/openvino-dev/))
+
+```
+omz_downloader --print_all
+```
+
+* Download the optimized Intermediate Representation (IR) of model (execute once), for example:
```

Comment:
It's time to make a model.lst config file, in order to simply the model opti/installation steps?

Reply:
The download_model.lst and convert_model.lst have been added under the data folder. The docker instruction has been updated.